### PR TITLE
doc: edit on github

### PR DESCRIPTION
**Pull request by noahdesu**

### CODE REVIEW

**Comment by noahdesu** [comment]
jenkins render docs THANKS

**Comment by noahdesu** [comment]
jenkins render docs THANKS

**Comment by noahdesu** [comment]
jenkins render docs THANKS

**Comment by noahdesu** [comment]
jenkins render docs or else

File: admin/build-doc [hunk 1]
```diff
@@ -124,6 +124,21 @@ for target in $sphinx_targets; do
...
+import json
+import yaml
+
+def json_serialize(obj):
+    if isinstance(obj, datetime.date):
+        return obj.isoformat()
+
+with open("$TOPDIR/doc/releases/releases.yml", 'r') as fp:
+    releases = yaml.load(fp)
+    print(json.dumps(releases, indent=2, default=json_serialize))
```
Comment (tchaikov) [inline admin/build-doc]: might want to add
```py
from __future__ import print_function
```

to be py2 compatible.

**Comment by noahdesu** [comment]
jenkins render docs THANKS

**Comment by noahdesu** [comment]
jenkins render docs

**Comment by noahdesu** [comment]
jenkins render docs

**Comment by noahdesu** [comment]
jenkins render docs

**Pull request merged**